ping.py

```diff
@@ -330,7 +331,7 @@ def _get_interface(remote, source):
             return interface
     return 'uplink'
 
-def ping(paths, duration_ms=1000, remotes=default_remotes, interface=None, verbosity='normal', address_type=None, ping_deadline = None):
```

Comment:
Dont think so, Ping  count is 1 ( -c param)

```
       -w deadline
           Specify a timeout, in seconds, before ping exits regardless of how many packets have been sent or received. In this case ping does not stop after count packet are sent, it waits either for
           deadline expire or until count probes are answered or for some error notification from network.

       -W timeout
           Time to wait for a response, in seconds. The option affects only timeout in absence of any responses, otherwise ping waits for two RTTs.
```

Reply:
Ahh now I understand...
Sorry
I think its would be neat to play around with the Timeout Time for some scenarios.
The default timeout is normaly 2 RTT

Reply:
We should be clear about deadline and timeout
Both is simmular but different

Reply:
Feel free to propose a solution. Or let's only focus on the foramtting for now.

Reply:
You are right.
I have things become terribly confused